opencsp/app/scene_reconstruction/test/test_SceneReconstruction.py

```diff
@@ -49,6 +50,12 @@ def setUpClass(cls, dir_input: str = None, dir_output: str = None):
             base_dir = dirname(__file__)
             dir_input = join(base_dir, 'data', 'data_measurement')
             dir_output = join(base_dir, 'data', 'data_expected')
+        ft.create_directories_if_necessary(dir_input)
+        ft.create_directories_if_necessary(dir_output)
+
+        path, _, _ = ft.path_components(__file__)
+        cls.out_dir = join(path, "data", "output", "ImageAcquisitionAbstract")
```

Comment:
Is the output directory supposed to be "ImageAcquisitionAbstract" for this unit test? Could this be a copy/paste error?

Reply:
Yeah, that's a copy paste error. Nice catch!